Update the software IO TLB filter rule to match new format

erickshepherdNI opened the pull request:
This change should fix the AzDO bug here: https://dev.azure.com/ni/DevCentral/_workitems/edit/2817929

The format of the "software IO TLB " dmesg line change from "software IO TLB: mapped mem..." to "software IO TLB: mapped [mem...". This change updates the filter pattern to reflect this.

erickshepherdNI commented:
I tested the change by running the kernel_dmesg_diff.py script locally and confirmed the software IO TLB message is no longer listed.

jpautler commented:
The change looks fine. There are a couple tweaks you could make to your commit message:

1. Change your commit message to include a "kernel_test:" prefix. This allows someone looking at the repo commit history to know at a glance what component was changed by the commit. See the following commit as an example:
https://github.com/ni/meta-nilrt/commit/10c39f850911a67317837a012a66e26791fc6385
1. Change your commit message to include a "Signed-off-by:" line at the bottom. See the commit above as an example. If you use the -s option to "git commit" it will do this automatically. Just make sure you have your email address set properly in your git configuration.

erickshepherdNI commented:
@chaitu236 Can you take a look at this PR? I don't think I have the permissions to add reviewers to the PR.

chaitu236 commented:
> @chaitu236 Can you take a look at this PR? I don't think I have the permissions to add reviewers to the PR.

Sure. The reason you can't add reviewers is you're not added to RTOS team. Can you get yourself added to NI org and then I can add you to RTOS team.

chaitu236 reviewed (changes requested):
The change looks fine but commit message needs to be signed (`git commit -s`). Also, first line of commit message needs to mention either filename or recipe name (and try to keep first line less than 72 chars if possible). The general OE guidelines are listed [here](https://docs.yoctoproject.org/dev/contributor-guide/submit-changes.html#implement-and-commit-changes) which we try to follow.

You can also refer to [history of this file](https://github.com/ni/meta-nilrt/commits/nilrt/master/kirkstone/recipes-kernel/kernel-tests/kernel-tests-files/kernel_dmesg_diff.py) and [other commits in this repo](https://github.com/ni/meta-nilrt/commits/nilrt/master/kirkstone/) for examples.

erickshepherdNI committed:
kernel_dmesg_diff.py: Update the software IO TLB filter rule.

The IO TLB filter message changed format and is no longer caught by the rule.
This change updates the rule to catch the new format.

Signed-off-by: Erick Shepherd <erick.shepherd@ni.com>

chaitu236 commented:
Cherry picked to nilrt/master/next - https://github.com/ni/meta-nilrt/commit/8b84d2d3db1bcde08cb45265fa811ab3dd223615
